app/src/routes/v1/permission.js

```diff
@@ -1,5 +1,6 @@
 const routes = require('express').Router();
 
+const { permissionValidator } = require('../../validators');
```

Comment:
Style: This should be introduced after the permissionController - have some degree of alphabetical ordering based on require path sorting.